pkg/parcacol/querier.go

```diff
@@ -1117,37 +1174,58 @@ func (q *Querier) selectMerge(
 	}
 
 	var valueCol logicalplan.Expr = logicalplan.Col(profile.ColumnValue)
-	if resultType.Type == "samples" && resultType.Unit == "count" {
-		valueCol = logicalplan.Mul(logicalplan.Col(profile.ColumnValue), logicalplan.Col(profile.ColumnPeriod)).Alias(profile.ColumnValue)
-		resultType = queryParts.Meta.PeriodType
```

Comment:
we needed this

Reply:
Yes, it's there. Just formatted.

Reply:
Ah, wait. You're talking about the `resultType = queryParts.Meta.PeriodType` specifically?